src/modules/utils.js

```diff
@@ -280,6 +280,14 @@ self = module.exports = {
             !url.href.match(/^https?:\/\/|^file:\/\//);
 
         return !!sameOrigin;
+    },
+
+    isNumber: function (value) {
+        if (typeof value === 'number') {
+            return true;
+        }
+
+        return !!value.match(/-?(\d+|\d+\.\d+|\.\d+)([eE][-+]?\d+)?/);
```

Comment:
I think there are better ways to check for a number. For example, this will fail with European decimal notation (which uses a comma, not a period)? Some alternatives (that could replace the entire method):

    return !isNaN(parseFloat(value)) && isFinite(value);

Or the way jQuery does it (pre 3.0):

    return !Array.IsArray(value) && (value - parseFloat(value) + 1) >= 0;

jQuery 3.x:

    var type = typeof value;
    return (type === "number" || type === "string") && !isNaN(value - parseFloat(value));